MSAL/src/native_auth/network/errors/sign_up/MSALNativeAuthSignUpContinueResponseError.swift

```diff
@@ -85,11 +127,22 @@ extension MSALNativeAuthSignUpContinueResponseError {
              .verificationRequired,
              .credentialRequired,
              .none:
-            return .init(type: .generalError, message: errorDescription)
+            return .init(
+                type: .generalError,
+                message: errorDescription,
+                correlationId: context.correlationId(),
+                errorCodes: errorCodes ?? [],
+                errorUri: errorURI
+            )
         }
     }
 
-    func toAttributesRequiredPublicError() -> AttributesRequiredError {
-        return AttributesRequiredError(message: errorDescription)
+    func toAttributesRequiredPublicError(context: MSIDRequestContext, message: String? = nil) -> AttributesRequiredError {
```

Comment:
why in this case we have the message as method parameter?

Reply:
Because the sign-up controller creates a custom message: 

> attribute_validation_failed from signup/continue submitAttributes request. Make sure these attributes are correct: attribute1, attribute2.